dependencies/CMakeLists.txt

```diff
@@ -13,92 +13,52 @@ elseif(UNIX)
   endif()
 endif()
 
+set(CPP_WORKER_SDK_HEADER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/worker_sdk/include")
```

Comment:
Traditionally, this is called `<library>_INCLUDE_DIR`, so perhaps call this variable `CPP_WORKER_SDK_INCLUDE_DIR`

Reply:
Changed